src/sql-parser/src/ast/defs/statement.rs

```diff
@@ -85,6 +85,7 @@ pub enum Statement<T: AstInfo> {
     Subscribe(SubscribeStatement<T>),
     ExplainPlan(ExplainPlanStatement<T>),
     ExplainTimestamp(ExplainTimestampStatement<T>),
+    ExplainSchema(ExplainSchemaStatement<T>),
```

Comment:
This is a bit of bike-shedding, but Schema is such an overloaded term is there a better name we could use? Maybe something like `ExplainSinkSchema`?

Reply:
I think `ExplainSchema` is fine though if you prefer that.